lib/inspec/resources/sys_info.rb

```diff
@@ -13,13 +13,38 @@ class System < Inspec.resource(1)
       describe sys_info do
         its('hostname') { should eq 'example.com' }
       end
+
+      describe sys_info do
+        its('hostname') { should eq 'example.com' }
```

Comment:
It looks like this example block is identical to the previous one - did you mean to edit it to include an option to `hostname`?

Reply:
Oops yeah meant to update that.